.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
Does it totally disable blank issue creation? Having it as a last resource option at the bottom was nice.

Or maybe we should have a template for it now?

Reply:
I'm fine with enabling it.